pkg/kubelet/images/image_gc_manager.go

```diff
@@ -171,30 +171,30 @@ func NewImageGCManager(runtime container.Runtime, statsProvider StatsProvider, r
 
 func (im *realImageGCManager) Start() {
 	go wait.Until(func() {
-		// Initial detection make detected time "unknown" in the past.
-		var ts time.Time
-		if im.initialized {
-			ts = time.Now()
-		}
-		_, err := im.detectImages(ts)
-		if err != nil {
-			klog.Warningf("[imageGCManager] Failed to monitor images: %v", err)
-		} else {
-			im.initialized = true
-		}
-	}, 5*time.Minute, wait.NeverStop)
-
-	// Start a goroutine periodically updates image cache.
-	// TODO(random-liu): Merge this with the previous loop.
-	go wait.Until(func() {
+		counter := 0
 		images, err := im.runtime.ListImages()
 		if err != nil {
 			klog.Warningf("[imageGCManager] Failed to update image list: %v", err)
 		} else {
 			im.imageCache.set(images)
 		}
-	}, 30*time.Second, wait.NeverStop)
+		counter++
```

Comment:
Nit: I wonder, is there any downside to running the `detectImages` component every 30 seconds? I'm slightly worried about the impacts of using the counter on readability.

If we do decide to keep the `counter%10`, can we add a comment (or better yet, descriptive variable/func name) indicating that `counter%10` equates to once every 5 minutes?

Reply:
```
any downside to running the detectImages component every 30 seconds
```
I conformed to the original period. I think we should be consistent across refactoring.

I found that I lost the branch where this PS was based on.

Please let me know whether I should open new PR for adding the comment, or the current PR is good to go - it is pretty obvious what the actual period is , dividing 10 by 2.

Thanks